tests/test_ble_runner.py

```
@@ -311,6 +311,72 @@ async def _noop():
             with runner._instance_lock:
                 runner._loop = original_loop
 
+    def test_run_coroutine_threadsafe_timeout_alias_warns_every_call(
+        self, monkeypatch
+    ):
+        """Legacy timeout alias is a semantic migration and should warn on each call."""
+        runner = BLECoroutineRunner()
+        monkeypatch.setattr(runner, "_ensure_running", lambda timeout=None: None)
```

Ruff (0.15.2)

[warning] Unused lambda argument: `timeout`

(ARG005)